src/app/static/templates/500.ftl

```diff
@@ -0,0 +1,11 @@
+<#include "partials/header.ftl">
+<div class="container mt-5">
+    <div class="row">
+        <div class="col-lg-6 offset-lg-3 col-10 offset-1">
+            <h2>Something Went Wrong</h2>
```

Comment:
Maybe we should, in future tickets, log the error and provide an id here as we do in HINT/VIMC